spring-integration-webflux/src/test/java/org/springframework/integration/webflux/outbound/WebFluxRequestExecutingMessageHandlerTests.java

```diff
@@ -214,6 +218,10 @@ void testFluxReply() {
 		reactiveHandler.setExpectedResponseType(String.class);
 		reactiveHandler.setReplyPayloadToFlux(true);
 
+		reactiveHandler.setAttributeVariablesExpression(new SpelExpressionParser().parseExpression("{name:{first:'Nikola',last:'Tesla'},dob:{day:10,month:'July',year:1856}}"));
```

Comment:
We prefer no more than `120` symbols in one code line.

Reply:
Also, doesn't look like you do some verification for those attributes to propagate.

Reply:
New test case has been added to check the propagation of attributes. Also code has been changes to keep less than 120 char in one line.